web-ifc-three/src/IFC/components/IFCManager.ts

```diff
@@ -197,9 +197,19 @@ export class IFCManager {
      * @scene Scene where the model is (if it's located in a scene).
      */
     close(modelID: number, scene?: Scene) {
-        this.state.api.CloseModel(modelID);
-        if (scene) scene.remove(this.state.models[modelID].mesh);
-        delete this.state.models[modelID];
+        try {
+            this.state.api.CloseModel(modelID);
+            const mesh = this.state.models[modelID].mesh;
+            const { geometry, material } = mesh;
+            if (scene) scene.remove(mesh);
+            geometry?.dispose();
+            geometry = null;
```

Comment:
![Screenshot (738)](https://user-images.githubusercontent.com/47943405/216075727-e8c0a633-ab09-4175-bee8-de1fb641a9d8.png)

Reply:
@D4ve-R  Getting this error on `npm run build`

Reply:
hi @aka-blackboots , sorry about that i was try to be extra correct and trigger the gb collector.
Kind of weird because geometry & material are inherited from threejs' Mesh class and should be overwriteable.
I fixed it by removing the assignment.

Reply:
No issues, let me look into this.